examples/tinywl/XdgStackWorkspace.qml

```diff
@@ -154,6 +155,34 @@ Item {
 
                     waylandSurface.setMaximize(false)
                 }
+
+                function onRequestMinimize() {
+                    if (waylandSurface.isResizeing)
+                        return
+
+                    if (waylandSurface.isMinimize)
+                        return
+
+                    focus = false;
+                    Helper.setActivateSurface = null;
```

Comment:
`activeSurface`? And `if (Helper.activeSurface == surface)` before `Helper.activeSurface = null`